src/components/HomeCard/index.js

```diff
@@ -0,0 +1,259 @@
+import React, { useState } from "react";
+import Slider from 'react-input-slider';
+import style from "./styles";
+
+
+const HomeCard = ({ className }) => {
+    const [valor, setState] = useState({ x: 0 });
+    let titles = ["No que podemos te ajudar?", "De quantos seguidores você precisa?", "De quantas curtidas você precisa?", "De quantas curtidas e seguidores você precisa?"];
+    let followers = ["1000", "1500", "3000", "6000"]
+    let likes = ["25", "50", "100", "200"]
+    let followersprices = ["55,90", "75,90", "145,90", "279,90"]
+    let likesprices = ["25,90", "45,90", "84,90", "159,90"]
+    let combosprices = ["89,90", "125,90", "249,90", "489,90"]
+    let linksfollower = ["http://mon.net.br/17s8xk", "http://mon.net.br/17s8xl", "http://mon.net.br/17s8xm", "http://mon.net.br/17s8xn"]
+    let linkslikes = ["http://mon.net.br/17s8xv", "http://mon.net.br/17s8xx", "http://mon.net.br/17s8y0", "http://mon.net.br/17s8y1"]
+    let linkscombos = ["http://mon.net.br/17s8y3", "http://mon.net.br/17s8y4", "http://mon.net.br/17s8y5", "http://mon.net.br/17s8y7"]
```

Comment:
Instead of having multiple arrays referring to different attributes of three products we could have an array of objects, each member being one product with its attributes.

Reply:
Fixed at 63f5f9b